source/features/linkify-branch-references.tsx

```diff
@@ -48,26 +37,30 @@ const hovercardObserver = new MutationObserver(([mutation]) => {
 			</a>,
 		);
 	}
-});
+}
 
-function hovercardInit(): void | Deinit {
-	const hovercardContainer = select('.js-hovercard-content > .Popover-message');
-	if (hovercardContainer) {
-		hovercardObserver.observe(hovercardContainer, {childList: true});
-		return hovercardObserver;
-	}
+async function init(signal: AbortSignal): Promise<void> {
+	observe('.branch-name', linkifyQuickPR, {signal});
+}
+
+function hovercardInit(signal: AbortSignal): void {
+	observe('[data-hydro-view*="pull-request-hovercard-hover"] ~ .d-flex.mt-2', linkifyHovercard, {signal});
 }
 
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isQuickPR,
-		pageDetect.isEditingFile,
```

Comment:
Already linked

https://github.com/refined-github/refined-github/edit/main/source/feature-manager.tsx

<img width="626" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/875709fc-b853-4650-9088-ddd3dcb50eb6">

Then there's the reference in the "commit" popup, but I don't care about it:

<img width="574" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/6a2412de-7fa4-4e47-a41c-a5b962a722ed">